NOTES.UNIX

```diff
@@ -5,38 +5,92 @@
  For Unix/POSIX runtime systems on Windows, please see NOTES.WIN.
 
 
- Shared libraries and installation in non-standard locations
- -----------------------------------------------------------
-
- Binaries on Unix variants expect to find shared libraries in standard
- locations, such as /usr/lib, /usr/local/lib and some other locations
- configured in the system (for example /etc/ld.so.conf on some systems).
- If the libraries are installed in non-standard locations, binaries
- will not find them and therefore fail to run unless they get a bit of
- help from a defined RPATH or RUNPATH.  This can be applied by adding
- the appropriate linker flags to the configuration command, such as
- this (/usr/local/ssl was the default location for OpenSSL installation
- in versions before 1.1.0):
+ Using compiler to link programs and shared libraries
+ ----------------------------------------------------
 
-    $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
-        -Wl,-rpath,/usr/local/ssl/lib
+ OpenSSL's generated Makefile uses the C compiler command line to
+ link programs, shared libraries and dynamically loadable shared
+ objects.  With this in mind, any linking option that's given to the
+ configuration scripts MUST be in a form that the compiler can accept.
+ This varies between systems, where some have compilers that accept
+ linker flags directly, while others take them in '-Wl,' form.  You need
+ to read your compiler documentation to figure out what it can take, and
+ ld(1) to figure out what linker options are available.
 
- Because the actual library location may vary further (for example on
- multilib installations), there is a convenience variable in Makefile
- that holds the exact installation directory and that can be used like
- this:
 
-    $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
-        -Wl,-rpath,'$(LIBRPATH)'
+ Shared libraries and installation in non-default locations
+ ----------------------------------------------------------
+
+ Binaries on some Unix variants expect to find shared libraries in
+ default locations, such as /lib, /usr/lib or possibly /usr/local/lib.
```

Comment:
I kind of have problems parsing this sentence. It sounds like binaries on some Unix variants do, while binaries on other Unix variants don't. When intention is likely to say that on some/most[?] Unix variants default locations are /lib, /usr/lib. BTW, I'd like to suggest to say "dynamically-linked binaries", not just "binaries".

Reply:
"Every Unix system has its own set of default locations for shared libraries" maybe?  And then delete lines 26-27 (the parenthetical comment).  And then fix the word-wrap

Reply:
I like @richsalz's suggestion.  Going with it.